test/parallel/test-vm-function-declaration.js

```diff
@@ -28,17 +28,19 @@ const o = vm.createContext({ console });
 
 // Function declaration and expression should both be copied to the
 // sandboxed context.
-let code = 'var a = function() {};\n';
+let code = 'let a = function() {};\n';
```

Comment:
This doesn’t seem like it should work because `let` does not create a global variable?